qframe_test.go

```diff
@@ -1373,6 +1373,65 @@ mon
 	})
 }
 
+func TestQFrame_ReadCSVMissingColumnName(t *testing.T) {
```

Comment:
These tests would probably lend themselves nicely to "table driven testing" if that's something you want to explore: https://github.com/golang/go/wiki/TableDrivenTests

Reply:
I think I'm going to have to spend some more time getting familiar with TableDrivenTests.

Might do another PR for documentation seprately if that's ok.

Reply:
If you have things that you know you'd like to do I'd prefer if we did it all in this PR to make it as complete. Otherwise I'm OK with the current state of affairs. Examples are great  but not a must.